dev-qt/qtdbus/qtdbus-5.9.2.ebuild

```diff
@@ -11,14 +11,20 @@ if [[ ${QT5_BUILD_TYPE} == release ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~x86"
 fi
 
-IUSE=""
+IUSE="examples"
```

Comment:
I'm also wondering if the eclass could add the flag to `IUSE` instead of doing it in every ebuild... How many modules do *not* have examples?

Reply:
I did it last time here: https://github.com/gentoo/qt/pull/153, you still didn't like it, did you?
And if "examples" USE-flag is generated, but used in REQUIRED_USE, QA bot might complain that it's not defined (at least it did for gentoo/qt repo).